Cargo.toml

```
@@ -120,6 +121,7 @@ sp-arithmetic = { version = "26.0.0", default-features = false }
 sp-api = { version = "35.0.0",  default-features = false }
 sp-blockchain = { version = "38.0.0", default-features = false }
 sp-block-builder = { version = "35.0.0", default-features = false }
+sp-consensus = { version = "=0.41.0" }
```

since we are tracking the cargo lock file, we don't need the exact match anymore, unless we know there is a bug in the minor/patch version
```suggestion
sp-consensus = { version = "0.41.0" }
```